ja-jp/content/topics/security/mfa/yubikey.htm

```diff
@@ -402,7 +402,7 @@ <h3><span class="mc-variable _3rd_Party_Integrations.YubiKey_company_name variab
                                                         <li value="1"><span class="mc-variable okta-variables.ProductName variable">Okta</span>にサインインします。</li>
                                                         <li value="2"><span class="mc-variable okta-feature-names.Sign-In_Widget variable">サインインウィジェット</span>の<span class="uicontrol">［Set up factors（要素の設定）］</span>ページで、<span class="uicontrol"><span class="mc-variable _3rd_Party_Integrations.YubiKey_company_name variable">［YubiKey］</span></span>の下の<span class="uicontrol">［Set up（設定）］</span>をクリックします。<span class="uicontrol">［Set up <span class="mc-variable _3rd_Party_Integrations.YubiKey_company_name variable">YubiKey（YubiKeyの設定）］</span></span>ページが表示されます。</li>
                                                         <li value="3"><span class="mc-variable _3rd_Party_Integrations.YubiKey_company_name variable">YubiKey</span>を挿入し、プロンプトが表示されたらボタンをタップします。</li>
-                                                        <li value="4"><span class="uicontrol">［Review（確認）］</span>をクリックします。<span class="uicontrol">［Set up security methods（セキュリティ方法の設定）］</span>ページが表示されます。</li>
+                                                        <li value="4"><span class="uicontrol">［Review（確認）］</span>をクリックします。<span class="uicontrol">［Set up security methods（セキュリティ方式の設定）］</span>ページが表示されます。</li>
```

Comment:
Interesting to see some of the text in braces changed to only Japanese (see line 396 above `[Delete]`), but some remained in English like this line. What is the criteria to translate or not translate?